mlflow/R/mlflow/R/model.R

```diff
@@ -28,8 +29,22 @@ mlflow_save_model <- function(model, path, ...) {
 #' @export
 mlflow_log_model <- function(model, artifact_path, ...) {
   temp_path <- fs::path_temp(artifact_path)
-  mlflow_save_model(model, path = temp_path, ...)
-  mlflow_log_artifact(path = temp_path, artifact_path = artifact_path)
+  model_spec <- mlflow_save_model(model, path = temp_path, model_spec = list(
+    utc_time_created = mlflow_timestamp(),
+    run_id = mlflow_get_active_run_id(),
+    artifact_path = artifact_path,
+    flavors = list()
+  ), ...)
+  res <- mlflow_log_artifact(path = temp_path, artifact_path = artifact_path)
+  mlflow_record_logged_model(model_spec)
```

Comment:
Should we try-catch this to handle the case where the server doesn't have the log-model endpoint implemented?

Reply:
yes, we should! good catch